app/controllers/quote_anonymous_controller.rb

```diff
@@ -0,0 +1,19 @@
+# Entry point for the quotes_anonymous resource
+class QuoteAnonymousController < ApplicationController
+  before_action :ilid?
+
+  def show
+    quote = Quote.find_by_id!(params[:id])
+
+    if quote.user_id != current_user.id && !current_user.admin?
+      flash.now[:error] = 'Je kan alleen je eigen quote anoniem maken'
+      return render turbo_stream: turbo_stream.update('flash', partial: 'layouts/alert')
+    end
+
+    quote.user_id = nil
+    quote.reporter_id = nil
+    quote.save!
+    quote.destroy_history
+    redirect_to quote_path(quote)
```

Comment:
In plaats van een redirect kunnen we een turbo frame gebruiken om hem meteen te laten zien. Lijkt een stuk smoother :)

Reply:
Geen flauw idee hoe dit allemaal werkt en ook iets minder zin om dit uit te zoeken 😅
Ik kwam tot `render turbo_stream: turbo_stream.update('quote', partial: 'quotes/quote', locals: { quote: quote })` maar dat wil niet

Reply:
Ik kan je wel een keer laten zien hoe zoiets mooi kan, laat het voor nu maar zo